packages/react/src/patterns/blocks/ContentBlockSimple/__stories__/ContentBlockSimple.stories.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import readme from '../README.md';
+import { settings } from 'carbon-components';
+import { storiesOf } from '@storybook/react';
+
+const { prefix } = settings;
+
+storiesOf('Patterns (Blocks)|ContentBlockSimple', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const mediaData = object('Media Data:', ContentGroupSimpleKnobs.mediaData);
+    const types = ContentGroupSimpleKnobs.types;
+    const mediaType = select('Media type:', types, types.image);
+    const items = object('Content Items:', ContentGroupSimpleKnobs.items);
+
+    const copy = {
+      'single paragraph': `Lorem    ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+      nulla quis, consequat libero. Here are
+      some common categories:`,
+      'multiple paragraphs': `   Lorem    ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+      nulla quis, consequat libero. Here are
+      some common categories:
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+      `,
+      'multiple paragraphs (styled)': `   __Lorem__    ipsum *dolor* sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+      Mauris iaculis eget dolor nec hendrerit. __Phasellus__ at elit sollicitudin, sodales
+      nulla quis, *consequat* libero. Here are
+      some common categories:
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+      `,
+    };
+
+    const ctaStyles = {
+      text: 'text',
+      card: 'card',
+      none: null,
+    };
+
+    const ctaTypes = {
+      external: 'external',
+      jump: 'jump',
+      local: 'local',
+    };
+
+    const ctaProps = {
+      style: select('CTA style', ctaStyles, ctaStyles.text),
+      type: select('CTA types', ctaTypes, ctaTypes.local),
+      title: 'Lorem ipsum dolor sit amet',
+      href: 'https://www.ibm.com',
+      copy: 'Lorem ipsum dolor sit ametttt',
+    };
+
+    return (
+      <div className={`${prefix}--grid`}>
```

Comment:
So you're missing a row and column. I'd suggest following the pattern we've been using throughout the rest of the patterns.

```html
<div class="bx--grid ">
  <div class="bx--row">
    <div class="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
      ...
    </div>
  </div>
</div>
```

Reply:
This goes back to the question of whether the columns should be dictated by authors or the component. I feel like it should be at the component level, at least for the largest size. @wonilsuhibm is this correct?

Reply:
It's dictated by the section patterns.